pkg/generate/app/sourcelookup.go

```diff
@@ -204,14 +204,17 @@ func (r *SourceRepository) Detect(d Detector, dockerStrategy bool) error {
 	if r.info != nil {
 		return nil
 	}
+
```

Comment:
+1, i'm not averse to introducing whitespace if you're already touching the file anyway, but it does tend to cross a line if you're not making any other changes.